src/main/java/mate/academy/springbookapp/service/impl/BookServiceImpl.java

```diff
@@ -53,4 +57,14 @@ public BookDto updateById(Long id, CreateBookRequestDto createBookRequestDto) {
     public void deleteById(Long id) {
         bookRepository.deleteById(id);
     }
+
+    @Override
+    public List<BookDto> search(BookSearchParametersDto bookSearchParametersDto) {
+        Specification<Book> bookSpecification
```

Comment:
This code can be a single method chain - there is no need to store `bookSpecification` variable, use `return` statement immediately.